dev/com.ibm.ws.jaxb_fat/publish/servers/com.ibm.ws.jaxb.tools.TestServer/bootstrap.properties

```diff
@@ -0,0 +1,2 @@
+com.ibm.ws.logging.trace.specification=ddmodel=all=enabled:*=info=enabled
```

Comment:
should ddmodel=all be enabled? We are trying to reduce as much trace as possible in our logs to save disk space.